docs/examples/general/data_loading/numpy_reader.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<p>For this tutorial I think that text headers would be better instead of setting "titles" in the code samples via comment. This applies to all the file specification methods as well as the different ROI flavors below.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>